src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -182,14 +187,13 @@ class AddressValidationModal extends React.Component {
       addressFromUser,
       validationKey,
       addressValidationError,
-      closeModal,
       resetAddressValidation,
       confirmedSuggestions,
     } = this.props;
 
     const resetDataAndCloseModal = () => {
       resetAddressValidation();
-      closeModal();
+      this.props.closeModal();
```

Comment:
for some reason, none of the dispatches to close the modal were showing up in redux dev tools - by switching to this pattern, they now do.